ui/src/dashboards/actions/thunks.ts

```diff
@@ -240,7 +240,7 @@ export const deleteDashboard = (dashboard: Dashboard) => async (
   try {
     const resp = await api.deleteDashboard({dashboardID: dashboard.id})
 
-    if (resp.data !== 204) {
+    if (resp.status !== 204) {
```

Comment:
why is the type for data in the generated routes set to `any`?
```
interface DeleteDashboardNoContentResult {
  status: 204
  headers: Headers
  data: any
}
```

Reply:
@121watts got any context on this?

Reply:
.. the swagger is defined as.. 
```
      responses:
        '204':
          description: Delete has been accepted
```
and the response actually comes back as 
```
status: 204
headers: {}
data: undefined
```

Reply:
but it looks like oats just creates an `any` data. that's annoying.. but not sure where it can be fixed.